library/lua/gui/dwarfmode.lua

```diff
@@ -426,6 +426,8 @@ function DwarfOverlay:simulateCursorMovement(keys, anchor)
 end
 
 function DwarfOverlay:onAboutToShow(parent)
+    DwarfOverlay.super.onAboutToShow(self, parent)
```

Comment:
this currently calls an empty function, but something *could* be there in the future.